docs/operations/README.md

```diff
@@ -54,9 +54,10 @@ If a bug fix or feature needs to be made to stable out of the normal cycle, this
 In the case where a bug fix or feature needs to be added outside of the normal cycle, the code-fix branch and release will be handled differently than normal:
 
 1. Code will need to be branched NOT off of main, but off of the same commit as the most recent stable commit. This should be the one tagged with the most recent vX.XX.XX value.
-2. After making the bug fix, the approved PR will branch will be tagged with a new release tag, incrementing the patch value from the last commit number.
-3. This branch then needs to be merged to main per the usual process.
-4. This same branch should be merged into staging.
+2. After making the bug fix, the approved PR branch will not be merged yet, instead it will be tagged with a new release tag, incrementing the patch value from the last commit number.
+3. If main and stable are on the the same commit then merge this branch into the staging using the staging release tag (staging-<the hotfix release number>). 
+4. If staging is already ahead stable, you may need to create another branch that is based off of the current staging commit, merge in your code change and then tag that branch with the staging release.
```

Comment:
**question (non-blocking)**: I'm trying to figure out what we get froma staging release of our hotfix. Is the idea that we will test the hotfix on staging with this release? Or do we make it after the fact so staging is more like stable? Since staging is going to be like the main of the future, do we want instead to make a staging release from `main` once the hotfix PR lands there?

Reply:
The idea would be that when its time for that staging code to be deployed to stable (most likely less than a week from the day the hotfix is done), it will have the hotfix code without having to redeploy main to staging and wait a week. It would be so that staging include the stable hotfix, so in that sense yes it is to make them more inline with eachother. The issues with just retagging main is then you still have to wait a full week from the time you did the retag in order to get that code on stable. (Unless we do away with that rule)